src/main/scala/io/opentargets/etl/backend/Evidence.scala

```diff
@@ -0,0 +1,142 @@
+package io.opentargets.etl.backend
+
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.spark.sql._
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.types._
+import spark.{Helpers => H}
+
+object Evidence extends LazyLogging {
+  val normaliseString: String => String = c => c.replaceAll("[ .-]", "_")
+  val toCamelCase: String => String = c => {
```

Comment:
I think Cinzia wrote a function in Helpers that does the same thing but on DataFrames. I think this method and `normaliseString` would be better there as we'll probably need them in several steps.

Reply:
Miguel and I already talked about this piece of code.
This is a manipulation of the current evidences index. 
The next step is to rewrite this step. So I am happy to refactor it later.

Reply:
@JarrodBaker @cmalangone we should try to decompose functions into simpler ones when it is possible and trying those to be composable in a way